.github/workflows/ci.yml

```diff
@@ -21,9 +21,7 @@ jobs:
         with:
           node-version: 18
 
-      - uses: pnpm/action-setup@v2
-        with:
-          version: 8.15
```

Comment:
Could you please the exact pnpm version? 

That's one thing with pnpm that's a bit troublesome: different versions (even minor versions) yield different lockfile structure, failing installations.

Reply:
I re-added the versions to match the version in `package.json`.

Reply:
Thanks!